src/odemis/driver/test/orsay_test.py

```diff
@@ -1823,7 +1864,7 @@ def setUpClass(cls):
         if TEST_NOHW == "sim":  # Actual stays 0 in simulation, so set it directly
             cls.ov.Actual = 10000
         else:
-            cls.fibbeam.objectiveVoltage.value = 10000
+            cls.fibbeam.objectiveVoltage.value = 14000
```

Comment:
Why 14kV? The comment just above says it's the same as the baseLensVoltage, but now it's not anymore. 
Also, why do you set the VA only in the real server? Could we always set this VA, and afterwards, in case it's the simulator, also change the `ov.Actual` to simulate the change of voltage. Probably this last command will update the VA, so it'd have the same behaviour, but that would be clearer.

Reply:
Ok. Then just update the comment to be correct, and it's fine :-)